src/services/__tests__/QueueService.test.ts

```diff
@@ -90,5 +90,25 @@ describe('Queue Service', () => {
 
       expect(actualUsers).toEqual([matchingUser1, matchingUser2, matchingUser3]);
     });
+
+    it('should return users that match some of the languages if nobody matches all languages', async () => {
+      const users: User[] = [nonMatchingUser1, matchingUser1, matchingUser3, matchingUser2];
+      userRepo.listAll = jest.fn().mockResolvedValueOnce(users);
+      const givenLanguages = ['Java', 'C#', 'Other'];
+
+      const actualUsers = await getInitialUsersForReview(givenLanguages, 2);
+
+      expect(actualUsers).toEqual([matchingUser1, matchingUser2]);
+    });
+
+    it('should not return any users if there are no matches', async () => {
+      const users: User[] = [nonMatchingUser1, matchingUser1, matchingUser3, matchingUser2];
```

Comment:
nitpick: This reads a little weird now since the "matchUser"s are actually not matching in this case.